src/main/java/org/folio/consortia/utils/HelperUtils.java

```diff
@@ -11,4 +14,17 @@ public static void checkIdenticalOrThrow(String firstString, String secondString
       throw new IllegalArgumentException(errorMsg);
     }
   }
+
+  public static boolean existingUserUpToDate(User user) {
```

Comment:
basically we need to rename: isUserActive(User user) or even we don't need this methid at all

Reply:
fixed